### DISCUSSION TITLE
native wireless kexts

### DISCUSSION THREAD

**Discussion by ivan-ef** [osy86/HaC-Mini#1071] [General]
Does open core blocking any native Apple kexts? 
Can't get my BCM94360CS2 modules to work on Monterey. Bluetooth is working, but wifi is not. 
No wifi module is showing in system report. No wifi module in hackintool peripherals list.

Should Apple native 80211family kext be loaded and showed in kextstat?

**Comment by SashonIT** [comment]
Maybe the Ending CS2 is then the wrong Card which works OOB.

I ordered from China a BCM94360 NG not CS2

Also you need to deactivate or deleting the previous Kext Files.

BluetoolFixup.kext 

Maybe it Helps when you load up your EFI and show it to me.... then i can have a look.

We have all the same Maschine.

Which OS you are Running?

Btw. is Apple Software Update running? 
![SoftwareUpdate](https://user-images.githubusercontent.com/126977959/227717429-59179f34-f3d5-4797-ac6d-76b2f7ec9f6b.png)

I mean this here --->

**Comment by kennx** [comment]
I have the same problem. I found that it's due to insufficient EFI partition space.

**Comment by ivan-ef** [comment]
The issue was a dead apple to nvme adapter.
